tests/loadtests/vkloadtests.cmake

```diff
@@ -11,40 +11,32 @@ else()
             # Fallback for inconsistent variable name on Windows
             set(Vulkan_INCLUDE_DIRS ${Vulkan_INCLUDE_DIR})
         endif()
+        set(VULKAN_SDK "${Vulkan_INCLUDE_DIR}/..")
```

Comment:
Why did you add this?

Reply:
`find_package(Vulkan)` only gives include and lib paths. This code also needs the share and Frameworks paths, so I need the Vulkan root path (SDK path).